core-bundle/src/EventListener/DataContainer/Undo/LabelListener.php

```diff
@@ -62,7 +62,11 @@ public function __invoke(array $row, string $label, DataContainer $dc): string
     private function getTemplateData(string $table, array $row, array $originalRow): array
     {
         $dataContainer = $this->framework->getAdapter(DataContainer::class)->getDriverForTable($table);
-        $originalDC = new $dataContainer($table);
+        $ref = new \ReflectionClass($dataContainer);
+        $originalDC = $ref->newInstanceWithoutConstructor();
+        $originalDC->table = $table;
```

Comment:
This should IMHO set the protected field `strTable` instead, because there is no magic setter for `table` in the `DataContainer` class. 

You can also see this with the currently failing 8.2 test:

> Creation of dynamic property Contao\CoreBundle\Fixtures\Contao\DC_NewsTableStub::$table is deprecated

Should we maybe also use `intId` and `objActiveRecord`? These feel more part of the API to me, but on the other hand with custom data containers can do pretty much anything…